appveyor.yml

```diff
@@ -15,8 +16,13 @@ branches:
     only:
       - master
 
+platform:
+  - x86
+
 install:
-  - ps: Install-Product node $env:nodejs_version
+  # Workaround from https://github.com/appveyor/ci/issues/2921#issuecomment-486727727
+  # https://www.appveyor.com/docs/lang/nodejs-iojs/#installing-any-version-of-nodejs-or-iojs
```

Comment:
One thing this shows is how out of date the AppVeyor dos are :(

They continue to reference Node.js v5.x and iojs 🤦🏼‍♂️